src/txdb.cpp

```diff
@@ -268,6 +269,18 @@ bool CCFundDB::GetPaymentRequestIndex(std::vector<CFund::CPaymentRequest>&vect)
     return true;
 }
 
+
+bool CCFundDB::WriteTipHeight(int nHeight) {
+    return Write(DB_PROP_TIP_HEIGHT, nHeight);
+}
+
+int CCFundDB::ReadTipHeight() {
+    int nHeight = 0;
+    if (!Read(DB_PROP_TIP_HEIGHT, nHeight))
+        return 0;
+    return nHeight;
```

Comment:
As far as I can tell this always turns 0. Is that was is meant to happen?

Reply:
it would return 0 if its not able to read the chain tip height into nHeight, otherwise it would return the stored value